src/test/java/api/support/fakes/processors/StorageRecordPreProcessors.java

```diff
@@ -0,0 +1,131 @@
+package api.support.fakes.processors;
+
+import static api.support.http.InterfaceUrls.holdingsStorageUrl;
+import static org.apache.commons.lang3.ObjectUtils.firstNonNull;
+import static org.folio.circulation.domain.representations.ItemProperties.EFFECTIVE_CALL_NUMBER_COMPONENTS;
+import static org.folio.circulation.domain.representations.ItemProperties.HOLDINGS_RECORD_ID;
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.circulation.domain.representations.ItemProperties;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseHandler;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import api.support.APITestContext;
+import io.vertx.core.json.JsonObject;
+
+public final class StorageRecordPreProcessors {
+  private static final Logger log = LoggerFactory.getLogger(StorageRecordPreProcessors.class);
+
+  private static final List<String> CALL_NUMBER_PROPERTIES = Arrays.asList(
+    "callNumber", "callNumberPrefix", "callNumberSuffix"
+  );
+
+  private StorageRecordPreProcessors() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  @SuppressWarnings("unused")
```

Comment:
Why is this unused?

Reply:
Inspections highlights that `oldItem` param is unused, but it is required because record pre-processor is a BiFunction.

Reply:
Can you apply the `SuppressWarnings("unused")` to the `oldItem` parameter instead? In this case, it looks like the whole method is unused, which is confusing.